renderer/index.js

```diff
@@ -99,6 +99,14 @@ function init () {
       }
     } else if (e.which === 32) { /* spacebar pauses or plays the video */
       dispatch('playPause')
+    } else if ((e.ctrlKey || e.metaKey) && e.altKey && e.which === 39) {
+      dispatch('playbackJump', state.playing.currentTime + 10)
+    } else if ((e.ctrlKey || e.metaKey) && e.altKey && e.which === 37) {
+      dispatch('playbackJump', state.playing.currentTime - 10)
+    } else if ((e.ctrlKey || e.metaKey) && (e.which === 107 || e.which === 171)) { /* CMD || CTRL + DOM_VK_ADD || DOM_VK_PLUS  */
+      dispatch('setPlaybackRate', state.playing.playbackRate + 10)
```

Comment:
**playbackRate** is the relative speed to normal so:

* 1.0 is normal speed
* 2.0 is double speed
* 0.5 is half speed (slow motion :D)

and the negative ones are for playing backwards so:
* -1.0 is backwards, normal speed
* -0.5 is backwards, half speed
* -2.0 is backwards, double speed